lightning/src/ln/peer_handler.rs

```diff
@@ -47,7 +47,7 @@ use bitcoin::hashes::{HashEngine, Hash};
 pub trait CustomMessageHandler: wire::CustomMessageReader {
 	/// Called with the message type that was received and the buffer to be read.
 	/// Can return a `MessageHandlingError` if the message could not be handled.
-	fn handle_custom_message(&self, msg: Self::CustomMessage) -> Result<(), LightningError>;
+	fn handle_custom_message(&self, msg: Self::CustomMessage, org: &PublicKey) -> Result<(), LightningError>;
```

Comment:
Can you rename org to something more descriptive like "sender_node_id"?

Reply:
Better indeed updated